src/Hedgehog.Experimental.CSharp.Tests/Usings.cs

```diff
@@ -0,0 +1 @@
+global using Xunit;
```

Comment:
Why this instead of `using Xunit;` in `GenTests.cs`?

Reply:
No reason, I just created a new test project and .NET (or whatever) created me this as a template. Perhaps this is the modern way in .NET/C#...